### PR TITLE
FkProducer send

### PR DESCRIPTION
**Pull request by h1alexbel**
## PR-Codex overview
This PR adds a `FkMetadataTask` and `FkProducer` classes to the `io.github.eocqrs.kafka.fake` package, providing a fake implementation of the Kafka producer. 

### Detailed summary
- Adds a `FkMetadataTask` class to the `io.github.eocqrs.kafka.fake` package, providing a fake implementation of the `FutureTask<RecordMetadata>` class.
- Adds a `FkProducer` class to the `io.github.eocqrs.kafka.fake` package, providing a fake implementation of the `Producer` interface.
- `FkProducer` class now throws an `IllegalArgumentException` if the topic does not exist.
- `FkProducer` class now writes data to the `FkBroker` when a message is sent.
- Adds a test case for the `FkMetadataTask` class.
- Adds test cases for the `FkProducer` class, testing that it creates a fake producer with a mock broker, creates a fake producer, closes without exception, sends a message without topic existence, sends a message, and reads metadata in the right format.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by zoeself** [comment]
@h1alexbel thank you for your Pull Request. I'll assign someone to review it soon. 

 If this PR solves a ``todo`` from the code, please don't forget to remove it.

**Comment by h1alexbel** [comment]
metadata tightly coupled with connection to the real broker..

**Comment by h1alexbel** [comment]
@l3r8yJ take a look, please

File: src/test/java/io/github/eocqrs/kafka/fake/FkProducerTest.java [hunk 1]
```diff
@@ -26,31 +26,137 @@
...
+    final int partition = 0;
+    final FkBroker after = this.broker
+      .with(new TopicDirs(topic).value());
+    final Producer<String, String> producer =
+      new FkProducer<>(
+        after
+      );
+    producer.send("test-key", new KfData<>(data, topic, partition));
+    MatcherAssert.assertThat(
+      "Message is send in right format",
```
Comment (l3r8yJ) [inline src/test/java/io/github/eocqrs/kafka/fake/FkProducerTest.java:123]:
```suggestion
      "Sent data is not blank",
```
maybe?

File: src/main/java/io/github/eocqrs/kafka/fake/FkProducer.java [hunk 2]
```diff
@@ -38,22 +42,73 @@
...
+          )
+      );
+    }
+    this.broker.with(new DatasetDirs<>(key, message).value());
+    final RecordMetadata metadata = new RecordMetadata(
+      new TopicPartition(
+        message.topic(),
+        message.partition()
+      ),
+      0L,
```
Comment (l3r8yJ) [inline src/main/java/io/github/eocqrs/kafka/fake/FkProducer.java:85]: magic numbers

**Review by l3r8yJ — CHANGES_REQUESTED** [review]
@h1alexbel take a look at my comments below, please

File: src/main/java/io/github/eocqrs/kafka/fake/FkProducer.java (hunk 2 above)
Comment (h1alexbel) [reply src/main/java/io/github/eocqrs/kafka/fake/FkProducer.java]: @l3r8yJ ready for review again

**Review by l3r8yJ — APPROVED** [review]
@h1alexbel lgtm!

**Comment by h1alexbel** [comment]
@rultor merge

**Pull request merged**